keyboards/atset/atr1/readme.md

```diff
@@ -0,0 +1,30 @@
+# ATSET AT1
+
+![ATSET AT1](https://imgur.com/fu0iXD0h.jpg)
+
+A 9% macropad.
+
+* Keyboard Maintainer: [atset]
+* Hardware Supported:
+   
+    * ATR1 R1 *(ft. STM32F103)*
+* Hardware Availability: 
```

Comment:
Please provide a link to where this keyboard can be purchased under the **Hardware Availability** section.
```suggestion
* Hardware Supported: ATR1 R1 PCB with STM32F103
* Hardware Availability: 
```